test/integration/kubernetes_deploy_test.rb

```diff
@@ -255,13 +319,15 @@ def test_deploy_fails_if_required_binding_not_present
   def test_long_running_deployment
     2.times do
       assert deploy_fixtures('long-running')
+      assert_logs_match(%r{Service/multi-replica\s+Selects 2 pods})
     end
 
     pods = kubeclient.get_pods(namespace: @namespace, label_selector: 'name=jobs,app=fixtures')
     assert_equal 4, pods.size
 
     count = count_by_revisions(pods)
     assert_equal [2, 2], count.values
+    assert_logs_match(%r{Service/multi-replica\s+Selects 4 pods})
```

Comment:
These new assertions (and the new service in this test case) are regression tests for https://github.com/Shopify/kubernetes-deploy/pull/116/files#diff-ae1efd633a69ca93e2ad9f56702555ebR153 (displaying stale info for the "final" status of resources that succeeded quickly, e.g. because the service was pulled out of the poll loop as soon as 1 replica existed, but it had 4 by the end of the deploy).